docs/rules/max-classes-per-file.md

```diff
@@ -0,0 +1,43 @@
+# enforce a maximum number of classes per file (max-classes-per-file)
+
+Ensures that each file has a single purpose and holds responsibility
```

Comment:
We generally like to keep this section focused on general principles/introduction to the concept the rule is trying to enforce, without talking about what the rule actually does. Could you step back in this paragraph and just focus on why one class per file is a best practice? (Basically, even if the beginning were transformed to something like "Ensuring that each file has a single purpose and holds responsibility for only one class is a best practice", only less wordy and more accessible, would get 95% of the way there. The second sentence in this paragraph is perfect.)